config/services.yaml

```diff
@@ -51,6 +51,10 @@ services:
         tags:
             - { name: kernel.event_listener, event: kernel.exception }
 
+    App\Command\TranskribusAuthCommand:
+        tags:
+            - { name: 'console.command', command: 'app:transkribus' }
```

Comment:
The command name could be defined in `TranskribusAuthCommand::configure()` instead of here (with `$this->setName( 'app:transkribus' )`). That keeps this file free of unnecessary things, and puts the definition closer to where it's used.